tests/Test.ADAL.NET.Common/TestConstants.cs

```diff
@@ -51,8 +53,58 @@ public class TestConstants
         public static readonly string ErrorSubCode = "ErrorSubCode";
         public static readonly string CloudAudienceUrnMicrosoft = "urn:federation:MicrosoftOnline";
         public static readonly string CloudAudienceUrn = "urn:federation:Blackforest";
+
+        public static string GetTokenHomeUrl()
```

Comment:
we should only have 1 function. the function should take the authority as a value like "https://login.microsoftonline.com/common", "https://login.microsoftonline.com/home" etc etc